dev-php/pecl-ast/metadata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+
+<pkgmetadata>
+  <maintainer type="person">
+    <email>alex@1stleg.com</email>
+    <name>Alex Barker</name>
+  </maintainer>
+  <maintainer type="project">
+    <email>proxy-maint@gentoo.org</email>
+    <name>Proxy Maintainers</name>
+  </maintainer>
+  <maintainer type="project">
+    <email>php-bugs@gentoo.org</email>
+    <name>PHP</name>
+  </maintainer>
```

Comment:
Have you talked to this project beforehand, do they want to share the maintenance of this package? You cant add maintainers without their consent.

Reply:
Sigh, I am just going to leave this here...  You all can decide what to do with it in a few years when someone gets to 703130. 👍

Reply:
Approved for proxy maint by mjo in #gentoo-php on 04/17/2020.  Bug update requested at that time.

Reply:
Proxy-maint will be fine for taking this, but you should remove PHP project, because apparently they don't want anything to do with this. 

Also mjo doesn't belong to either projects, so I don't know what his approval does here...